src/enkf/CMakeLists.txt

```diff
@@ -6,8 +6,8 @@ if(BUILD_ENKF)
   set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
 
   if(BUILD_PRODUCTION)
-    set(ENKF_SUFFIX "gfs")
```

Comment:
I don't think we want this in the GSI master

Reply:
Changed.